source/fundamentals/crud/run-command.txt

```diff
@@ -0,0 +1,159 @@
+.. _golang-run-command:
+
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a raw **database command** with the
+{+driver-short+}. You can use database commands to perform a variety of
+administrative and diagnostic tasks, such as fetching server statistics,
+initializing a replica set, or running an aggregation pipeline.
+
+Compose a Command
+-----------------
+
+The {+driver-short+} provides two methods to run a database command. The
+``RunCommand()`` method executes the given command against the database
+and returns a ``SingleResult`` type. The ``RunCommandCursor()`` method
+executes the given command against the database and parses the response
+as a cursor.
+
+.. note:: Read Preference
+
+   These methods do not obey the read preference set for the database.
+   You can set a read preference for a database command by passing a
+   options to either method:
+   
+   .. code-block:: go
+
+      opts := options.RunCmd().SetReadPreference(readpref.Primary())
+      err := db.RunCommand(context.TODO(), command, opts).Decode(&result)
+   
+   For more information on
+   read preference options, see the :ref:`golang-write-read-pref`
+   fundamentals page.
+
+Each method takes a Context and command document. The
```

Comment:
Q: Is the term "a Context" well enough understood by Go devs to not require clarification here? Is it a code word that should be in monospace?

Reply:
I followed the formatting used in the [Context fundamentals page](https://www.mongodb.com/docs/drivers/go/current/fundamentals/context/). In that page, Context is captialized but not monospaced. It's a proper noun?